src/components/MessageBox.js

```diff
@@ -0,0 +1,96 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    expandable: React.PropTypes.bool,
+    icon: React.PropTypes.string,
+    title: React.PropTypes.string
+  },
+
+  getInitialState () {
+    return {
+      isOpen: false
+    };
+  },
+
+  _toggleMessageBox () {
+    this.setState({
+      isOpen: !this.state.isOpen
+    });
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div className='mx-message-box' style={styles.component}>
+        <div onClick={this._toggleMessageBox} style={styles.header}>
+          <div style={styles.leftHeader}>
+            <Icon
+              elementProps={{
+                onClick: this._toggleMessageBox
```

Comment:
Is the `onClick` handler necessary here since the parent div on line 32 has the same handler?